cdap-ui/app/features/admin/controllers/NamespaceDatasetMetadataController.js

```diff
@@ -1,15 +1,27 @@
 angular.module(PKG.name + '.feature.admin').controller('AdminNamespaceDatasetMetadataController',
```

Comment:
We can't do that, because if it ever throw a 404 error, it will show up in myAlert, since 404 error is handled globally.